day_6/exercises/burrito.rb

```diff
@@ -1,19 +1,35 @@
-# Add the following methods to this burrito class and 
+# Add the following methods to this burrito class and
 # call the methods below the class:
 # 1. add_topping
 # 2. remove_topping
 # 3. change_protein
 
 class Burrito
-  attr_reader :protein, :base, :toppings
+  attr_accessor :protein, :base, :toppings
+
   def initialize(protein, base, toppings)
     @protein  = protein
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(toppings)
+    @toppings += toppings
+  end
+
+  def remove_topping(toppings)
+    @toppings -= toppings
+  end
+
+  def change_protein(protein)
+    @protein = protein
+  end
 end
 
 dinner = Burrito.new("Beans", "Rice", ["cheese", "salsa", "guacamole"])
 p dinner.protein
 p dinner.base
 p dinner.toppings
+p dinner.add_topping(["sour cream"])
```

Comment:
How could you rewrite the method above so that you can pass a string instead of an array here?

Reply:
I think you are talking about line 33. If so, you could implement the shovel operator on the string "sour cream":

`p dinner.add_topping(["sour cream"])`
to: 
`p dinner.toppings << "sour cream"`

I updated this file with this change as well :)